coderadar-ui/src/app/service/project.service.ts

```diff
@@ -251,6 +252,9 @@ export class ProjectService {
     return this.httpClient.post(this.apiURL + 'projects/' + id + '/analyze/reset', {}, {observe: 'response'}).toPromise();
   }
 
+  getContributorsForProject(projectId: number) : Promise<HttpResponse<Contributor[]>> {
+    return this.httpClient.get<Contributor[]>(this.apiURL + 'projects/' + projectId + '/contributors', {observe: 'response'}).toPromise();
+
```

Comment:
Missing closing bracket (`}`). The UI does not compile like this

Reply:
Inserted.